openshift-start-configured-cluster

```diff
@@ -7,10 +7,14 @@ source $DIR/config
 
 OC_CONFIG=`mktemp -d --suffix=.openshift-config`
 
-cd $GOPATH/src/github.com/openshift/origin
-export PATH="$( source hack/lib/init.sh; echo "${OS_OUTPUT_BINPATH}/$( os::build::host_platform )/" ):${PATH}"
+if [ ! -f oc ]; then
+   wget -N -L https://github.com/openshift/origin/releases/download/v3.10.0-rc.0/openshift-origin-client-tools-v3.10.0-rc.0-c20e215-linux-64bit.tar.gz
```

Comment:
We should use latest probably instead of a specific rc version, it will not age well

Reply:
To be honest I couldn't figure out any URL that does this. Would you please give me some pointers how to download the latest `oc`?

Reply:
Oh I am not sure, I just build openshift locally ... point is for latest you better use latest oc or at some point the oldest one will probably break.
In general I think it is ok to expect a user to have the oc binary available and in the path if they want to run openshift, no need for our script to pull oc